### DISCUSSION TITLE
UI / UX: Post colors and emphasis

### DISCUSSION THREAD

**Discussion by Steffo99** [cheeaun/phanpy#757] [Q&A] [answered]
I'm trying to figure out the design decisions behind certain things in the project to suggest or develop UI / UX changes.

I've been thinking about post colors / backgrounds, which, if I understand correctly, are currently as such:

- *no background* → new post
- ***yellow** gradient background* → new **reply**
- ***purple** gradient background* → **boosted** post
- ***yellow and purple** gradient background* → **boosted reply**
- ***cyan** gradient background* → followed **hashtag**
- ***green** gradient background* → **group** (???)

---

I love the association between colors and meanings, and so I love the gradients, but I was wondering if that doesn't emphasize colored replies and boosts over plain new posts, which usually are the most important to look at in a timeline...

Is the emphasis on replies and boosts and hashtags and groups deliberate?

Is there anything that should be done to balance or prioritize new posts over the other types?

Should perhaps background colors be selected differently instead?

---

[explicitly pinging @cheeaun as the project's main designer and developer since i don't think discussions trigger a notification otherwise]

**Comment by cheeaun** [comment, marked as answer]
Initially I was thinking of using (non-gradient) plain background for every post type, e.g. light blue background for replies and plain background for other posts.

The idea becomes more important for private mentions and got prioritised due to confusion from users when they see it on their timeline. So, striped-line background was introduced instead of plain background to give a sense of "danger" (or warning, e.g. don't post your password etc, they're not entirely secure or E2EE)

I've tried plain full-background colors in the early development of Phanpy and found that there's too much emphasis, as what you mentioned above 😉. If the background is yellow and the post is huge (many lines of sentences), it'll be a huge chunk of yellow box on the screen.

It took me quiet a while to "fix" this, found this gradient background idea from another site, tried it and in my opinion, seems to work for now (and the past 2 years). So, the goal of the gradient background is not to distract or emphasize the post, but just a small visual "hint" to tell users that this post is a little different. Original posts don't have gradient backgrounds or any hints as I suspect users are already used to it, migrating from all other social media sites. Instead of them being emphasized or deemphasized, think of it like a table with thousand of rows and every odd row has a gray background so that they're easier to scan.

The gradient background should be a sub-conscious thing. If they start to distract you from the original posts, you're probably too conscious about them now 😬

But, this **depends** on your timeline. If you follow only hashtags and not following anyone, your timeline will be full of cyan backgrounds 😂

> cyan gradient background → followed hashtag
> green gradient background → group (???)

- Yes, accounts can be groups https://docs.joinmastodon.org/entities/Account/#group
- Two more:
  - **yellow and cyan** gradient background → followed **hashtag** + **reply**
  - **yellow and green** gradient background → **group** + **reply**
